toucan_connectors/one_drive/one_drive_connector.py

```diff
@@ -233,6 +238,13 @@ def _retrieve_data(self, data_source: OneDriveDataSource) -> pd.DataFrame:
 
             df_all = df_all.append(df_current)
 
+        for date_col in data_source.parse_dates:
+            df_all[date_col] = pd.to_datetime(
+                (df_all[date_col] * 24 * 60 * 60).astype(int),
+                origin=pd.Timestamp('1899-12-30'),
+                unit='s',
+            )
```

Comment:
We need a try-except here, because the user can provide columns that are not necessary on type date !